tools/ci_build/github/android/nnapi_supported_ops.md

```diff
@@ -39,10 +43,11 @@ Keep in sync with doco generated from /docs/execution-providers/NNAPI-ExecutionP
 |ai.onnx:Slice||
 |ai.onnx:Softmax||
 |ai.onnx:Sqrt||
-|ai.onnx:Squeeze||
+|ai.onnx:Squeeze|Input axes must be constant.|
```

Comment:
nit: "should" for consistency with other notes. sorry for the other wording in the original comment.
```suggestion
|ai.onnx:Squeeze|Input axes should be constant.|
```